scylla-cdc-base/src/main/java/com/scylladb/cdc/model/worker/Worker.java

```diff
@@ -148,13 +148,6 @@ private void performActionsUntilStopRequested(Collection<TaskAction> actions) {
         } catch (InterruptedException e) {
             logger.atWarning().log("Worker interrupted");
         }
-
-        // Stop all "sleeping" futures.
-        try {
-            workerConfiguration.delayedFutureService.stop();
-        } catch (InterruptedException e) {
```

Comment:
You changed the behaviour here. Previously both shutdown and awaitTermination was called and now only shutdown will be called from `stop()`. @avelanarius is that ok? Previously we were waiting for 1000ms for all delayed tasks to finish. Now we may not wait at all or wait only 50ms.

Reply:
I think it is OK. I tested this code for quite a while and didn't see any problems with shutdown.